bindings/nodejs/index.d.ts

```diff
@@ -196,7 +196,7 @@ export declare class Desktop {
    * @param {string} url - The URL to open.
    * @param {string} [browser] - The browser to use.
    */
-  openUrl(url: string, browser?: string | undefined | null): void
+  openUrl(url: string, browser?: string | undefined | null): Element
```

Comment:
FYI Pretty sure this is an auto-generated file and `main` was not up-to-date from something previously merged?